src/component/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link, NavLink } from 'react-router-dom';
+import '../index.css';
+
+const Navbar = ({ title, routes }) => (
+  <nav className="nav-bar">
+    <Link className="nav-brand" to="/">{title}</Link>
+    <ul className="nav-links">
+      {routes.map(({ name, path }) => (
+        <li key={path}>
+          <NavLink className="nav-link" activeClassName="active-link" exact to={path}>{name}</NavLink>
+        </li>
+      ))}
+    </ul>
+    <IconButton name="person" />
```

Comment:
- [ ] Your profile icon should look like this 👇🏽 
![image](https://github.com/theresetuyi/React-Bookstore/assets/63473584/ff48057a-c830-4925-aed8-26e75aa3ecee)
This will ensure you are consistent with the design